components/tidb_query/src/rpn_expr/impl_encryption.rs

```diff
@@ -80,6 +81,21 @@ pub fn uncompressed_length(ctx: &mut EvalContext, arg: &Option<Bytes>) -> Result
         }
     }))
 }
+#[rpn_fn(capture = [ctx])]
+#[inline]
+pub fn random_bytes(ctx: &mut EvalContext, arg: &Option<Int>) -> Result<Option<Bytes>> {
+    match arg {
+        Some(arg) => {
+            if *arg < 1 || *arg > 1024 {
```

Comment:
Can you please define a constant for `1024`?

Reply:
refactored.